libs/utils/src/lib/helpers/hooks.ts

```diff
@@ -37,3 +37,5 @@ export const useWindowDimensions = () => {
 
   return windowDimensions;
 };
+
+export const useConsoleLog = (value: any) => useEffect(() => console.log(value), [value]);
```

Comment:
I just use this during development, not intended to ever commit a usage but helpful to have available.